addons/test_event_full/tests/test_website_event.py

```diff
@@ -0,0 +1,118 @@
+
+        view_model_exhibitor.write({
+            'website_meta_title': "exhibitors",
+        })
+
+        self.assertTrue(view_model_exhibitor.website_meta_title, "Exhibitors page should have a meta title after writing")
+        self.assertNotEqual(event.website_meta_title, view_model_exhibitor.website_meta_title, "Event and Exhibitors page should have different meta titles")
+
+        # SEO test for 'Community' page
+        view_model_community = self.env['ir.ui.view'].search([('name', '=', 'Community TestEvent')])
+        self.assertFalse(view_model_community.website_meta_title, "Community page should initially have no meta title")
+
+        view_model_community.write({
+            'website_meta_title': "community",
+        })
+
+        self.assertTrue(view_model_community.website_meta_title, "Community page should have a meta title after writing")
+        self.assertNotEqual(event.website_meta_title, view_model_community.website_meta_title, "Event and Community page should have different meta titles")
+
+        # SEO test for 'Talks' page
+        view_model_track = self.env['ir.ui.view'].search([('name', '=', 'Talks TestEvent')])
+        self.assertFalse(view_model_track.website_meta_title, "Talks page should initially have no meta title")
+
+        view_model_track.write({
+            'website_meta_title': "talks",
+        })
+
+        self.assertTrue(view_model_track.website_meta_title, "Talks page should have a meta title after writing")
+        self.assertNotEqual(event.website_meta_title, view_model_track.website_meta_title, "Event and Talks page should have different meta titles")
+
+        # SEO test for 'Agenda' page
+        view_model_agenda = self.env['ir.ui.view'].search([('name', '=', 'Agenda TestEvent')])
+        self.assertFalse(view_model_agenda.website_meta_title, "Agenda page should initially have no meta title")
+
+        view_model_agenda.write({
+            'website_meta_title': "agenda",
+        })
+
+        self.assertTrue(view_model_agenda.website_meta_title, "Agenda page should have a meta title after writing")
+        self.assertNotEqual(event.website_meta_title, view_model_agenda.website_meta_title, "Event and Agenda page should have different meta titles")
+
+        # SEO test for 'Talks Proposals' page
+        view_model_track_proposal = self.env['ir.ui.view'].search([('name', '=', 'Talk Proposals TestEvent')])
+        self.assertFalse(view_model_track_proposal.website_meta_title, "Talk Proposals page should initially have no meta title")
+
+        view_model_track_proposal.write({
+            'website_meta_title': "talk proposals",
+        })
+
+        self.assertTrue(view_model_track_proposal.website_meta_title, "Talk Proposals page should have a meta title after writing")
+        self.assertNotEqual(event.website_meta_title, view_model_track_proposal.website_meta_title, "Event and Talk Proposals page should have different meta titles")
+
```

Comment:
Can we make a helper function `check_seo_data` and write all the common steps in that function and then just call it?

Reply:
Hello @thsh-odoo 
Updated the test case. Can you please have a look at it.
Thanks.. 😄